src/org.xtuml.bp.integrity/.externalToolBuilders/Build.launch

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.ui.externaltools.ProgramBuilderLaunchConfigurationType">
+<stringAttribute key="org.eclipse.debug.core.ATTR_REFRESH_SCOPE" value="${workspace}"/>
+<mapAttribute key="org.eclipse.debug.core.environmentVariables">
+<mapEntry key="INCLUDE_TESTS" value="true"/>
+<mapEntry key="bp_install_dir" value="${eclipse_home}"/>
+</mapAttribute>
+<booleanAttribute key="org.eclipse.debug.ui.ATTR_LAUNCH_IN_BACKGROUND" value="false"/>
+<stringAttribute key="org.eclipse.ui.externaltools.ATTR_LOCATION" value="${system_path:mvn}"/>
+<stringAttribute key="org.eclipse.ui.externaltools.ATTR_RUN_BUILD_KINDS" value="full,incremental,auto,"/>
+<stringAttribute key="org.eclipse.ui.externaltools.ATTR_TOOL_ARGUMENTS" value="-Dtycho.disableP2Mirrors=true -Dmaven.test.failure.ignore=true -DskipTests=true -f ${git_dir:org.xtuml.bp.releng.parent}/../releng/org.xtuml.bp.releng.parent/pom.xml -pl org.xtuml.bp:org.xtuml.bp.docgen install"/>
```

Comment:
I see org.xtuml.bp.docgen in this line.  I'm guessing that should be org.xtuml.bp.integrity.  This makes me thing we should do a search inside this plugin for "bp.docgen" and see if there are any other places.